installation/scripts/installer.sh

```diff
@@ -97,6 +97,24 @@ if [ $KNATIVE ]; then
     kubectl -n kyma-installer patch configmap installation-config-overrides -p '{"data": {"global.knative": "true", "global.kymaEventBus": "false", "global.natsStreaming.clusterID": "knative-nats-streaming"}}'
 fi
 
+if [ $LOCAL ]; then
```

Comment:
please move override to the `installer-config-local.yaml.tpl` and sed replacing its value with `${MINIKUBE_IP}`